docs/qa/CometBFT-QA-38.md

```diff
@@ -0,0 +1,187 @@
+---
+order: 1
+parent:
+  title: CometBFT QA Results v0.38.x
+  description: This is a report on the results obtained when running CometBFT v0.38.x on testnets
+  order: 5
+---
+
+# CometBFT QA Results v0.38.x
+
+This iteration of the QA was run on CometBFT `v0.38.0-alpha.2`, the second
+`v0.38.x` version from the CometBFT repository.
+
+The changes with respect to the baseline, `v0.37.0-alpha.3` from Feb 21, 2023,
+include the introduction of the `FinalizeBlock` method to complete the full
+range of ABCI++ functionality (ABCI 2.0), and other several improvements
+described in the
+[CHANGELOG](https://github.com/cometbft/cometbft/blob/v0.38.0-alpha.2/CHANGELOG.md).
+
+## Testbed
+
+As in other iterations of our QA process, we have used a 200-node network as
+testbed, plus nodes to introduce load and collect metrics.
+
+### Saturation point
+
+As in previous iterations of our QA experiments, we first find the transaction
+load on which the system begins to show a degraded performance. Then we run the
+experiments with the system subjected to a load slightly under the saturation
+point. 
+
+The method to identify the saturation point is explained
+[here](CometBFT-QA-34.md#saturation-point) and its application to the baseline
+is described [here](TMCore-QA-37.md#finding-the-saturation-point). 
+
+We chose `c=1,r=400` as the configuration for our experiments, where `c` is the
+number of connections created by the load runner process to the target node, and
+`r` is the rate or number of transactions issued per second. We could have
+choosen `c=2,r=200`, which has a transaction load equal to `c=1,r=400`.
+
+## Latencies
+
+The following figure plots the latencies of the experiment carried out with the
+configuration `c=1,r=400`.
+
+![latency-1-400](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png).
+
+For reference, the following figure shows the latencies of one of the
+experiments in the baseline, where `c=2,r=200` is equivalent to the same
+configuration used in this experiment. As can be seen, latencies are quite
+similar, and even in some cases the baseline has slightly higher latencies.
+
+![latency-2-200-37](img37/200nodes_cmt037/e_75cb89a8-f876-4698-82f3-8aaab0b361af.png)
```

Comment:
We should put a note saying that the results should just be in the same ballpark to be considered "equivalent" here.

Reply:
Done.